src/SpeedyTransforms/SpeedyTransforms.jl

```diff
@@ -9,11 +9,16 @@ import FFTW
 import GenericFFT
 import LinearAlgebra
 import Primes
+import Adapt: adapt
 
 # SPEEDYWEATHER MODULES
 using ..LowerTriangularMatrices
 using ..RingGrids
 
+# import SpeedyWeatherCUDAExt
+using SpeedyWeather
+Base.get_extension(SpeedyWeather, :SpeedyWeatherCUDAExt)
```

Comment:
Ouh why is that needed? Wouldn't that load CUDA everytime SpeedyTransforms is loaded, i.e. everytime SpeedyWeather is loaded?

Reply:
Accidental commit, will roll back.